ote_sdk/ote_sdk/usecases/exportable_code/demo/requirements.txt

```diff
@@ -0,0 +1,3 @@
+openvino==2021.4
```

Comment:
I ended up having openvino==2021.4.0 installed in the environment. Is it fine?

Reply:
update to openvino==2021.4.2